src/ColorUtils.php

```diff
@@ -2,63 +2,79 @@
 
 namespace CommandString\Utils;
 
+use LengthException;
+
 class ColorUtils
 {
+    public const BLACK = "#FFFFFF";
+    public const WHITE     = "#000000";
+
     public static function hexToRgb(string $hex): array
     {
         $hex = str_replace("#", "", $hex);
-        if (strlen($hex) == 3) {
-            $r = hexdec(substr($hex, 0, 1) . substr($hex, 0, 1));
-            $g = hexdec(substr($hex, 1, 1) . substr($hex, 1, 1));
-            $b = hexdec(substr($hex, 2, 1) . substr($hex, 2, 1));
-        } else {
-            $r = hexdec(substr($hex, 0, 2));
-            $g = hexdec(substr($hex, 2, 2));
-            $b = hexdec(substr($hex, 4, 2));
+        $length = strlen($hex);
+
+        if ($length == 3) {
+            return array_map(
+                static fn (string $part): int => hexdec($part . $part),
+                str_split($hex)
+            );
+        }
+
+        if ($length == 6) {
+            return array_map(
+                static fn (string $part): int => hexdec($part),
+                str_split($hex, 2)
+            );
         }
-        return [$r, $g, $b];
+
+        throw new LengthException('Color hex must be either 3 or 6 characters long.');
     }
 
     public static function rgbToHex(int $r, int $g, int $b, $withPrefix = true): string
     {
-        $r = dechex($r);
-        $g = dechex($g);
-        $b = dechex($b);
-
-        $r = strlen($r) == 1 ? "0" . $r : $r;
-        $g = strlen($g) == 1 ? "0" . $g : $g;
-        $b = strlen($b) == 1 ? "0" . $b : $b;
+        $hex = array_map(
+            static fn (int $part): string => str_pad(
+                dechex($part),
+                2,
+                '0',
+                STR_PAD_LEFT
+            ),
+            [$r, $g, $b]
+        );
```

Comment:
```suggestion
        $hex = sprintf("%02x%02x%02x", $r, $g, $b);
```
Perhaps we could do this as well?